challenge1/styles.css

```
@@ -15,14 +14,114 @@ html {
     font-family: "Ubuntu", sans-serif;
 }
 
+header {
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+    padding: 1rem 2rem;
+    gap:2rem
```

Not very important but try to be consistent:

```suggestion
    justify-content: space-between;
    align-items: center;
    padding: 1rem 2rem;
    gap: 2rem
```